specification/assets/gherkin/evaluation.feature

```diff
@@ -57,11 +70,31 @@ Feature: Flag evaluation
 
   # errors
   Scenario: Flag not found
-    When a non-existent string flag with key "missing-flag" is evaluated with details and a default value "uh-oh"
+    When a hook is added to the client
+    And a non-existent string flag with key "missing-flag" is evaluated with details and a default value "uh-oh"
```

Comment:
i do not think we need to deviate from the ` Given a Boolean-flag with key "metadata-flag" and a default value "true"` format, if the flag is missing, we know that from the key, but i do not think we need to add the `non-existent` part